modules/vaos/app/controllers/vaos/application_controller.rb

```diff
@@ -2,5 +2,16 @@
 
 module VAOS
   class ApplicationController < ::ApplicationController
+    before_action :authorize
```

Comment:
Do we need the custom authorize method, how about `before_action { authorize :vaos, :access? }`

Reply:
on second thought -- I rather like raising a custom exception.